docs/resources/logs_custom_destination.md

```diff
@@ -0,0 +1,135 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_logs_custom_destination Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Logs Custom Destination API resource, which is used to create and manage Datadog log forwarding.
+---
+
+# datadog_logs_custom_destination (Resource)
+
+Provides a Datadog Logs Custom Destination API resource, which is used to create and manage Datadog log forwarding.
+
+## Example Usage
+
+```terraform
+resource "datadog_logs_custom_destination" "sample_destination" {
+  name    = "sample destination"
+  query   = "service:my-service"
+  enabled = true
+
+  http_destination {
+    endpoint = "https://example.org"
+    basic_auth {
+      username = "my-username"
+      password = "my-password"
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The custom destination name.
+
+### Optional
+
+- `elasticsearch_destination` (Block List) The Elasticsearch destination. (see [below for nested schema](#nestedblock--elasticsearch_destination))
```

Comment:
```suggestion
- `elasticsearch_destination` (Block List) The Elasticsearch destination. (See [below for nested schema](#nestedblock--elasticsearch_destination).)
```

Reply:
The `see [below for nested schema]` part is from the automatic documentation generation. Not something I can change.